dspace-api/src/test/java/org/dspace/eperson/GroupTest.java

```diff
@@ -206,6 +206,9 @@ public void findAllNameSort() throws SQLException {
         List<String> names = new ArrayList<>();
         List<String> sortedNames = new ArrayList<>();
         for (Group group : groups) {
+            if (group.getName() == null) {
+                continue;
```

Comment:
One problem *may* actually be in the `init()` section which creates a bunch of unnamed groups (these likely should all have dummy names to begin with).  https://github.com/Georgetown-University-Libraries/DSpace/blob/ebf256caa1f03194cb547dae7eada64428a1b3e2/dspace-api/src/test/java/org/dspace/eperson/GroupTest.java#L56

There may still be other methods creating unnamed groups as well.